tests/drivers/adc/adc_api/boards/nucleo_wb55rg.overlay

```diff
@@ -28,7 +28,7 @@
 		reg = <4>;
 		zephyr,gain = "ADC_GAIN_1";
 		zephyr,reference = "ADC_REF_INTERNAL";
-		zephyr,acquisition-time = <ADC_ACQ_TIME_MAX>;
+		zephyr,acquisition-time = <ADC_ACQ_TIME(ADC_ACQ_TIME_TICKS, 248)>;
```

Comment:
If this setting is made purely for testing a specific case, please add a comment about it.

Reply:
Done